src/components/pages/signUp/index.tsx

```diff
@@ -45,95 +52,90 @@ export default function SignUpPage(): ReactElement {
         <HStack style={{ height: '100%' }}>
             <VStack style={{ flex: 1, padding: '32px', overflowY: 'scroll' }}>
                 <VStack style={{ gap: '32px', width: '100%', maxWidth: '800px' }}>
-                    <HStack style={{ width: '100%' }}>
+                    <HStack style={{ width: '100%', maxWidth: '400px' }}>
```

Comment:
Instead of changing the maxWidth on every child, you could also just apply it to the parent :)

```suggestion
                <VStack style={{ gap: '32px', width: '100%', maxWidth: '400px' }}>
                    <HStack style={{ width: '100%' }}>
```